nodes/ui_chart.html

```diff
@@ -169,6 +185,22 @@
         <input type="text" id="node-input-cutout" style="width:35px"> %
         </span>
     </div>
+    <div class="form-row" id="ui-chart-colours">
```

Comment:
chart.js generates automatically. 9 fits nicely in the node panel and matches with the 3 per row as with gauges. Happy to adjust if you like :)